### PR TITLE
build: Address deprecation warnings/errors

### PR DESCRIPTION
**Pull request by palinek**

### CODE REVIEW

File: plugin-colorpicker/colorpicker.cpp [hunk 1]
```diff
@@ -56,7 +56,7 @@ ColorPickerWidget::ColorPickerWidget(QWidget *parent):
     QFrame(parent)
 {
     QFontMetrics fm (mLineEdit.font());
-    mLineEdit.setFixedWidth ( 10*fm.width (QStringLiteral("a")) );
+    mLineEdit.setFixedWidth ( 10*fm.horizontalAdvance (QStringLiteral("a")) );
```
Comment (tsujan) [inline plugin-colorpicker/colorpicker.cpp:59]: I think it's the time for bumping `REQUIRED_QT_VERSION` to the last LTS, 5.12. `QFontMetrics::horizontalAdvance()` doesn't exist in 5.10.

Comment (palinek) [reply plugin-colorpicker/colorpicker.cpp:59]: Damn it... :(

Comment (tsujan) [reply plugin-colorpicker/colorpicker.cpp:59]: IMHO, this is good but 5.10 is too old.

Comment (palinek) [reply plugin-colorpicker/colorpicker.cpp:59]: OK. Bumped the required Qt version.

**Review by tsujan — APPROVED** [review]
Let's assume Qt ≥ 5.12 and get rid of warnings.

**Pull request merged**